hibernate-reactive-core/src/main/java/org/hibernate/reactive/mutiny/Mutiny.java

```diff
@@ -871,6 +872,177 @@
 		boolean isOpen();
 	}
 
+	/**
+	 * A non-blocking counterpart to the Hibernate
+	 * {@link org.hibernate.StatelessSession} interface, which provides a
+	 * command-oriented API for performing bulk operations against a database.
+	 * <p/>
+	 * A stateless session does not implement a first-level cache nor interact
+	 * with any second-level cache, nor does it implement transactional
+	 * write-behind or automatic dirty checking, nor do operations cascade to
+	 * associated instances. Collections are ignored by a stateless session.
+	 * Operations performed via a stateless session bypass Hibernate's event
+	 * model and interceptors. Stateless sessions are vulnerable to data
+	 * aliasing effects, due to the lack of a first-level cache.
+	 * <p/>
+	 * For certain kinds of transactions, a stateless session may perform slightly
```

Comment:
I understand what you mean, but perhaps I'd say "operations" rather than "transactions" ?

Reply:
I'm really talking about a unit of work, not a single operation.

Reply:
yes I understand. I suspect most users will read "transactions" as literal transactions, and might think a stateless session won't help for non-transactionl operations (while we actually mean to suggest that it could be helpful either way)

Reply:
I've improved the Javadoc.